libsolidity/interface/StandardCompiler.cpp

```diff
@@ -1164,8 +1180,102 @@ std::map<std::string, Json::Value> StandardCompiler::parseAstFromInput(StringMap
 	return sourceJsons;
 }
 
+Json::Value StandardCompiler::importEVMAssembly(StandardCompiler::InputsAndSettings _inputsAndSettings)
+{
+	solAssert(_inputsAndSettings.language == "EVMAssembly");
+	solAssert(_inputsAndSettings.sources.empty());
+	solAssert(_inputsAndSettings.jsonSources.size() == 1);
```

Comment:
I don't think you actually have validations against this. I suspect these asserts will fail if you try to use more than one `assemblyJson` or if you include some Solidity, Yul or AST sources as well.

Reply:
I see you added a check against multiple `assemblyJson` but what about the other cases I mentioned? E.g. having both `assemblyJson` and `content`?

Reply:
No, the keccak256 field is not supported. I added a simple validation that basically, if EVMAssembly was set, all sources need to have one member named `assemblyJson` any other member is causing an error.